dev-python/confuse/confuse-1.0.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} pypy3 )
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
```

Comment:
This results in a test failure,
```
======================================================================
FAIL: test_dir_from_environ (test.test_paths.WindowsTestCases)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/confuse-1.0.0/work/confuse-1.0.0/test/test_paths.py", line 97, in test_dir_from_environ
    self.assertEqual(confuse.config_dirs(),
AssertionError: Lists differ: ['\\var\\tmp\\portage\\dev-python\\confuse-[129 chars]fig'] != ['C:\\Users\\test\\AppData\\Roaming', 'C:\\[19 chars]fig']

First differing element 0:
'\\var\\tmp\\portage\\dev-python\\confuse-[43 chars]ming'
'C:\\Users\\test\\AppData\\Roaming'

+ ['C:\\Users\\test\\AppData\\Roaming', 'C:\\Users\\test\\winconfig']
- ['\\var\\tmp\\portage\\dev-python\\confuse-1.0.0\\work\\confuse-1.0.0\\~\\AppData\\Roaming',
-  '\\var\\tmp\\portage\\dev-python\\confuse-1.0.0\\work\\confuse-1.0.0\\~\\winconfig']

======================================================================
FAIL: test_fallback_dir (test.test_paths.WindowsTestCases)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/confuse-1.0.0/work/confuse-1.0.0/test/test_paths.py", line 103, in test_fallback_dir
    self.assertEqual(confuse.config_dirs(),
AssertionError: Lists differ: ['\\var\\tmp\\portage\\dev-python\\confuse-[44 chars]ing'] != ['C:\\Users\\test\\AppData\\Roaming']

First differing element 0:
'\\var\\tmp\\portage\\dev-python\\confuse-[43 chars]ming'
'C:\\Users\\test\\AppData\\Roaming'

- ['\\var\\tmp\\portage\\dev-python\\confuse-1.0.0\\work\\confuse-1.0.0\\~\\AppData\\Roaming']
+ ['C:\\Users\\test\\AppData\\Roaming']

----------------------------------------------------------------------
Ran 197 tests in 0.126s
```
Since the test phase passes with python-3.6 and python-3.7 it really seems they don't have 3.8 support yet upstream.